### ISSUE TITLE
Unable to watch Subscription content like Apple TV

### ISSUE THREAD

**Issue by RisenCrypto** [ungoogled-software/ungoogled-chromium#2647]
### OS/Platform

Windows

### Installed

https://ungoogled-software.github.io/ungoogled-chromium-binaries/

### Version

Version 120.0.6099.129 (Official Build, ungoogled-chromium) (64-bit)

### Have you tested that this is not an upstream issue or an issue with your configuration?

- [X] I have tried reproducing this issue in Chrome and it could not be reproduced there
- [ ] I have tried reproducing this issue in vanilla Chromium and it could not be reproduced there
- [ ] I have tried reproducing this issue in ungoogled-chromium with a new and empty profile using `--user-data-dir` command line argument and it could not be reproduced there

### Description

I am unable to watch Apple TV.

### How to Reproduce?

1) Go to [Apple TV](https://tv.apple.com/in/)
2) Login
3) Try to watch any movie/TV show
You get the following error.

![Apple](https://github.com/ungoogled-software/ungoogled-chromium/assets/79957149/e21ce64f-77be-498c-989f-f7e3f159308f)

### Actual behaviour

![Apple](https://github.com/ungoogled-software/ungoogled-chromium/assets/79957149/e21ce64f-77be-498c-989f-f7e3f159308f)

### Expected behaviour

Video should play fine.

**Comment by y0LAs** [comment]
Same issue here, with Netflix, Disney+ and Prime Video.
There is a plugin with instructions for Linux users ↓, but nothing for Windows users, as far as I could research.
https://github.com/proprietary/chromium-widevine

**Comment by PF4Public** [comment]
Have you consulted [faq#how-do-i-install-widevine-cdm](https://ungoogled-software.github.io/ungoogled-chromium-wiki/faq#how-do-i-install-widevine-cdm)?

**Comment by y0LAs** [comment]
Yes, and I did it again after your response, but couldn't find any working solution.
For Windows, it leads to https://chromium.woolyss.com/#widevine, where it says that you have to install the Widevine plugin yourself, but on the official website, you can't download it anymore : https://www.widevine.com/download/videooptimizer Broken link, no downloads anymore on the website.
On https://chromium.woolyss.com/#widevine, it also says that you need to go to chrome://components/ and check that "Widevine Content Decryption Module" is an other version than 0.0.0.0.
But it is on Ungoogled Chromium, and clicking on "Chek for update" and waiting doesn't work.

Did you manage to make it work on Windows?

**Comment by RisenCrypto** [comment]
Looks like the widevine at https://dl.google.com/widevine-cdm/4.10.2710.0-win-x64.zip is a very old one & manifest gives errors when I uploaded it to ungoogled-chromium.

Unrecognized manifest key 'arch'.
Unrecognized manifest key 'os'.
Unrecognized manifest key 'x-cdm-codecs'.
Unrecognized manifest key 'x-cdm-host-versions'.
Unrecognized manifest key 'x-cdm-interface-versions'.
Unrecognized manifest key 'x-cdm-module-versions'.
Unrecognized manifest key 'x-cdm-persistent-license-support'.
Manifest version 2 is deprecated, and support will be removed in 2023. See https://developer.ch40me.qjz9zk/blog/mv2-transition/ for more details.
